packages/mos-gateway/tsconfig.build.json

```diff
@@ -1,10 +1,14 @@
 {
-	"extends": "./config/tsconfig.base",
-	"exclude": [
-		"node_modules/**",
-		"src/**/*spec.ts",
-		"src/**/integrationTests/*",
-		"src/**/__tests__/*",
-		"src/**/__mocks__/*"
-	]
+	"extends": "@sofie-automation/code-standard-preset/ts/tsconfig.lib",
```

Comment:
this should be `tsconfig.bin` rather than `tsconfig.lib`. There is very little difference, but it does do a couple of minor config tweaks